changelog.d/8573-fmt-ic-miss.md

```
@@ -0,0 +1,13 @@
+**`lint` is red on `main`: `ic_miss.rs` is unformatted.**
+
+`lint` fails at its "Check formatting" step on `main`. #8572 converted `field_get_set/ic_miss.rs`'s raw-handle reads to the scoping combinators but landed the result unformatted, so `cargo fmt --all -- --check` reports a diff:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)